lib/routes/sicau/jk.ts

```diff
@@ -128,27 +82,31 @@ export const route: Route = {
             headers: { 'x-access-token': token },
             method: 'post',
         });
-
-        const { code, message, content } = await $post(`/getUserSchoolActList`, {
-            query: {
-                gid: gidDict[gid],
-                typeId: typeDict[typeId],
-                sortType,
-                page: 1,
-            },
-        });
-
-        if (code !== '0') {
-            throw new Error(message);
+        const query = async (page: number) =>
+            await $post(`/getUserSchoolActList`, {
+                query: {
+                    gid: gidDict[gid],
+                    typeId: typeDict[typeId],
+                    sortType,
+                    page,
+                },
+            });
+
+        const res1 = await query(1);
+        const res2 = await query(2);
```

Comment:
Please parse only page 1.

Reply:
There is not enough data on the page 1, it doesn't cover all the newly released activities.

Reply:
Please perform the requests in parallel with `Promise.all()` instead of in series.

Reply:
OK